js/interface.js

```diff
@@ -1595,6 +1601,22 @@ $('body').on('click', '[data-rule-delete]', function (event) {
   updateDataSourceRules();
 });
 
+$('body').on('click', '[data-toggle-status]', function (event) {
+  event.preventDefault();
+
+  var index = parseInt($(this).closest('tr').data('rule-index'), 10);
+  var rule = currentDataSourceRules[index];
+
+  rule.enabled = !rule.enabled;
+
+  // Briefly show a UI feedback as the rule enables/disables
+  $(this).find('i')
+    .addClass('fa-spinner fa-animated fa-pulse')
```

Comment:
`fa-animated` isn't a supported class as far as I see on https://fontawesome.com/v4.7.0/examples/

It's `fa-spin` or `fa-pulse`. And for the `fa-spinner` icon you chose, `fa-pulse` is more appropriate than `fa-spin`.

```suggestion
    .addClass('fa-spinner fa-pulse')
```

Reply:
It wasn't working for me with just the pulse one, but I have refreshed and it's now working just with pulse. 🤷 not sure what happened but I fixed it now.